SVGKitFrameworks/SVGKitFramework-iOS/SVGKitFramework-iOS-Prefix.pch

```diff
@@ -1,30 +0,0 @@
-//
```

Comment:
Why are all these .h and prefix headers deleted?

Reply:
Each target share the same prefix header. The content are the same. So we can use a universal prefix header instead.

The same thing apply to the umbrella header (Which import all public headers), instead of copy files again and again. See the first commit.

Reply:
Ah, right. I have a suspicion that there was some reason for needing the separate headers - some bug or "feature" in xcode versions, or some problem to do with the build-for-app-store phase, where it got confused if you used a shared header.

I might be being paranoid here ;), but unless someone can test this change with a push-to-asset-store release, I'd prefer to shift it to a separate patch, and do that on a side branch and have people use it and wait until soneone's confirmed they got iphone, ipad, etc builds to go fine.

(i.e. don't let it slow down / poison the rest of this patch - if it's merely a code-cleanup thing?)